common/sgx/endorsements.c

```diff
@@ -334,8 +360,8 @@ oe_result_t oe_get_sgx_endorsements(
     size_t* endorsements_buffer_size)
 {
     oe_result_t result = OE_UNEXPECTED;
```

Comment:
How long does it take to execute oe_get_sgx_endorsements()?

Reply:
973200 microseconds (almost 1 second)

Reply:
I think you mean milliseconds instead of microseconds.  We need to optimize this.